src/main/java/com/incognia/CardType.java

```diff
@@ -0,0 +1,19 @@
+package com.incognia;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum CardType {
```

Comment:
What about naming this "PaymentType"? Although currently we only support credit and debit cards, it should be extensible to other payment methods.

Reply:
done